R/indentation_linter.R

```diff
@@ -313,15 +309,12 @@ indentation_linter <- function(indent = 2L, hanging_indent_style = c("tidy", "al
 }
 
 find_new_indent <- function(current_indent, change_type, indent, hanging_indent) {
-  if (change_type == "suppress") {
-    current_indent
-  } else if (change_type == "hanging") {
-    hanging_indent
-  } else if (change_type == "double") {
-    current_indent + 2L * indent
-  } else {
+  switch(change_type,
+    suppress = current_indent,
+    hanging = hanging_indent,
+    double = current_indent + 2L * indent,
     current_indent + indent
```

Comment:
That's `block =` then.

Reply:
Can you add this? I think it improves clarity to specify all cases.

Reply:
Oh, I think I added it to the wrong branch maybe... done now